static/library/utils.ts

```diff
@@ -17,14 +17,24 @@
 /** Library of helper functions shared across web components */
 
 /**
- * Custom attribute converter for Array<String> type attributes
- * Models the behavior of the default Lit attribute value converter, but
- * removes the requirement to use double quotes within a list.
+ * Custom attribute converter for Array<String> type attributes.
+ * Parses:
+ *   (1) A comma or space separated list of values
+ *   (2) A JSON formatted list of values
+ * into a Array<String> of values.
+ *
  * @param attributeValue the attribute value provided to the web component
- * @returns JSON parsed list of strings
+ * @returns A string array of attribute values
  */
 export function convertArrayAttribute(attributeValue: string): string[] {
-  // JSON.parse requires double quotes for strings within objects.
-  const cleanedAttributeValue = attributeValue.replaceAll("'", '"');
-  return JSON.parse(cleanedAttributeValue);
+  const openingBrackets = ["(", "[", "{"];
```

Comment:
Just use `[` here - valid json can't start with `(`, and an object starting with `{` won't have the return type of `string[]`

Reply:
Good call out, switched to just `[`.